### PR TITLE
Add parquet unit test for case insensitive column lookup

### PR DESCRIPTION
**Pull request by nezihyigitbasi**
Should be merged after #4686.

### CODE REVIEW

File: presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java [hunk 1]
```diff
@@ -251,6 +253,31 @@ public void testParquet()
...
+
+        HiveOutputFormat<?, ?> outputFormat = new MapredParquetOutputFormat();
+        InputFormat<?, ?> inputFormat = new MapredParquetInputFormat();
+        @SuppressWarnings("deprecation")
+        SerDe serde = new ParquetHiveSerDe();
+        File file = File.createTempFile("presto_test", "parquet");
+        file.delete();
+        try {
+            FileSplit split = createTestFile(file.getAbsolutePath(), outputFormat, serde, null, writeColumns, NUM_ROWS);
+            //use parquet column-based access
```
Comment (zhenxiao) [inline presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java]: column name based access?

**Comment by billonahill** [comment]
+1

**Comment by nezihyigitbasi** [comment]
Tests are expected to fail before #4686 is merged.

**Comment by dain** [comment]
Merged, thanks!

**Pull request closed without merging**